Extensions/classic_tags.js

```diff
@@ -82,6 +83,97 @@ XKit.extensions.classic_tags = new Object({
 		$(this).attr("href", newHref);
 	},
 
+	get_post_timestamp: function(blog_name, post_id) {
+		var self = this;
+		var api_url = "https://api.tumblr.com/v2/blog/" + blog_name + "/posts" + "?api_key=" + self.apiKey + "&id=" + post_id;
+		var promise = $.Deferred();
+
+		function fail() {
+			XKit.console.add('Unable to fetch timestamp for post ' + post_id);
```

Comment:
Either expose the error to the user (bad in this case I think) or just use
console.log.
On Sun, Nov 1, 2015 at 8:05 PM Milan Roberson <notifications@github.com>
wrote:

> In Extensions/classic_tags.js
> <https://github.com/new-xkit/XKit/pull/704#discussion_r43591520>:
>
> > @@ -82,6 +83,97 @@ XKit.extensions.classic_tags = new Object({
> >  		$(this).attr("href", newHref);
> >  	},
> >
> > +	get_post_timestamp: function(blog_name, post_id) {
> > +		var self = this;
> > +		var api_url = "https://api.tumblr.com/v2/blog/" + blog_name + "/posts" + "?api_key=" + self.apiKey + "&id=" + post_id;
> > +		var promise = $.Deferred();
> > +
> > +		function fail() {
> > +			XKit.console.add('Unable to fetch timestamp for post ' + post_id);
>
> Please don't use XKit.console.add, perhaps XKit.notifications.add(message,
> type, sticky, callback) instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/704/files#r43591520>.
>